malaria24/ona/models.py

```diff
@@ -89,6 +98,23 @@ def age(self):
         dob = datetime.strptime(self.date_of_birth, '%y%m%d')
         return int((today - dob).days / 365)
 
+    def get_ehps(self):
+        return Actor.objects.ehps().filter(facility_code=self.facility_code)
+
+    def get_email_context(self):
+        return {
+            'case': self,
+            'ehps': self.get_ehps(),
+        }
+
+    def get_text_email_content(self):
+        return render_to_string(
+            'ona/text_email.txt', self.get_email_context())
+
+    def get_html_email_content(self):
+        return render_to_string(
+            'ona/html_email.html', self.get_email_context())
+
```

Comment:
Yup! So I don't have to duplicate the code from `tasks.py` to display the email in the admin.